citi-onboarding/DADe-onepage
# Feature/view-organizacao-interna

## What I did

- Created organization view and several of its components

| Mockup | Real |
|--------|------|
![mockup real](https://user-images.githubusercontent.com/61971624/122129839-1361b800-ce0d-11eb-92b3-90604ce91f56.png)|![mockup real](https://user-images.githubusercontent.com/61971624/122129839-1361b800-ce0d-11eb-92b3-90604ce91f56.png) |   |

## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   1. `git checkout develop`
   2. `git pull`
   3. `git checkout feature/view-organizacao-interna`
   4. `git pull origin feature/view-organizacao-interna`
   5. `npm install`
   6. `cd client/`
   7. `npm start`

3. In your browser go to http://localhost:3000 (should open automatically when running `npm start`)

4. Check if the component (Organization) looks just like the mockup in different screen sizes (minimum width: 1366px) and if when you increase it past 1920px it remains the same size (Doesnt grow along with the screen).

5. In VScode open the `client\src\components\Card-Father` file

6. In lines 20 to 23 of the mentioned file, change the cardImage and cardText properties to any image or text, save the file, and check if the cards are displaying your changes correctly.

7. Don't commit or push your changes from the tests, as the original informations were the correct ones for the current version of the project. 

8. Test the things that you think are worth testing, even the ones that are not in this description
